.circleci/config.yml

```diff
@@ -1,7 +1,7 @@
 defaults: &defaults
   working_directory: ~/app/
   docker:
-    - image: cimg/node:14.19
+    - image: cimg/node:16.20.0
```

Comment:
Tiny: potentially moving to node 18

Reply:
Thanks Dima, let's do them all together as part of node 18 migration or node 20 if we can